### PR TITLE
OCPBUGS-30316: Add description to several IBU spec fields

### PR DESCRIPTION
**Pull request by leo8a**
/cc @donpenney @jc-rh @browsell @Missxiaoguo

### CODE REVIEW

File: api/v1alpha1/types.go [hunk 1]
```diff
@@ -67,31 +68,47 @@ type ImageBasedUpgradeSpec struct {
 	SeedImageRef SeedImageRef `json:"seedImageRef,omitempty"`
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Additional Images"
 	AdditionalImages ConfigMapRef `json:"additionalImages,omitempty"`
+	// OADPContent defines the list of ConfigMap resources that contain the OADP Backup and Restore CRs.
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="OADP Content"
 	OADPContent []ConfigMapRef `json:"oadpContent,omitempty"`
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Extra Manifests"
+	// ExtraManifests defines the ConfigMap resource for the user-specific extra manifests in the Prep stage.
```
Comment (Missxiaoguo) [inline api/v1alpha1/types.go:75]: Perhaps a more precise description?
```suggestion
	// ExtraManifests defines the list of ConfigMap resources that contain the user-specific extra manifests to be applied during the upgrade post-pivot stage .
```

Comment (leo8a) [reply api/v1alpha1/types.go:75]: sure, added the text enhancement to #399

**Pull request merged**